addons/io_scene_gltf2/blender/imp/gltf2_blender_map_occlusion.py

```diff
@@ -32,12 +33,75 @@ def create_nodetree(gltf, material_idx, vertex_color):
+        node_tree = material.node_tree
+
         BlenderTextureInfo.create(gltf, pymaterial.occlusion_texture.index)
 
-        # Pack texture, but doesn't use it for now. Occlusion is calculated from Cycles.
+        # Pack texture. Occlusion is calculated from Cycles or Eevee, so do nothing
         if gltf.data.images[gltf.data.textures[
             pymaterial.occlusion_texture.index
         ].source].blender_image_name is not None:
             bpy.data.images[gltf.data.images[gltf.data.textures[
                 pymaterial.occlusion_texture.index
             ].source].blender_image_name].use_fake_user = True
+
+        # Create a fake node group for exporter
+        gltf_node_group_name = get_gltf_node_name()
+        if gltf_node_group_name in bpy.data.node_groups:
+            gltf_node_group = bpy.data.node_groups[gltf_node_group_name]
+        else:
+            # Create a new node group
+            gltf_node_group = bpy.data.node_groups.new(gltf_node_group_name, 'ShaderNodeTree')
+            gltf_node_group.inputs.new("NodeSocketFloat", "Occlusion")
+            gltf_node_group.nodes.new('NodeGroupOutput')
+            gltf_node_group_input = gltf_node_group.nodes.new('NodeGroupInput')
+            gltf_node_group_input.location = -200, 0
+
+        # Set the node group inside material node tree
+        ao_node = node_tree.nodes.new('ShaderNodeGroup')
+        ao_node.node_tree = gltf_node_group
+        ao_node.location = 0, 200
+
+        # Check if the texture node already exists (if used by other parameter metal / roughness)
+        found = False
+        for node in [node for node in node_tree.nodes if node.type == "TEX_IMAGE"]:
+            if gltf.data.images[gltf.data.textures[
+                pymaterial.occlusion_texture.index
+            ].source].blender_image_name == node.image.name:
+                # This is our image !
+                # Retrieve separate node if found
+                if node.outputs['Color'].is_linked:
+                    for link in node.outputs['Color'].links:
+                        if link.to_node.type == 'SEPRGB':
+                            node_tree.links.new(ao_node.inputs[0], link.to_node.outputs[0])
+                            found = True
+                            break
+
+                if found == True:
+                    break
+
+        if found == False:
+            # Need to create the texture node & separate node
+            mapping = node_tree.nodes.new('ShaderNodeMapping')
+            uvmap = node_tree.nodes.new('ShaderNodeUVMap')
+            separate = node_tree.nodes.new('ShaderNodeSeparateRGB')
+            if pymaterial.occlusion_texture.tex_coord is not None:
+                uvmap["gltf2_texcoord"] = pymaterial.occlusion_texture.tex_coord  # Set custom flag to retrieve TexCoord
+            else:
+                uvmap["gltf2_texcoord"] = 0  # TODO set in pre_compute instead of here
```

Comment:
Remove/resolve TODO?

Reply:
This is a global TODO (I mean that this is not the only line of code with this TODO about initialisation of texture coordinate. I think we can merge with this TODO, and it will be removed when I will process it on entire source code of the addon